ui/app/common/attributeTypes/directives/attributeTypes.js

```diff
@@ -15,13 +15,23 @@ angular.module('bahmni.common.attributeTypes', []).directive('attributeTypes', [
         },
         templateUrl: '../common/attributeTypes/views/attributeInformation.html',
         restrict: 'E',
-        controller: function ($scope) {
+        controller: function ($scope, $log) {
             $scope.getAutoCompleteList = $scope.getAutoCompleteList();
             $scope.getDataResults = $scope.getDataResults();
             // to avoid watchers in one way binding
-            $scope.isAutoComplete = $scope.isAutoComplete() || function () { return false; };
-            $scope.isReadOnly = $scope.isReadOnly() || function () { return false; };
-            $scope.handleUpdate = $scope.handleUpdate() || function () { return false; };
+            $scope.isAutoComplete = $scope.isAutoComplete() || function () {
+                return false;
+            };
+            $scope.isReadOnly = $scope.isReadOnly() || function () {
+                return false;
```

Comment:
Change these lines to as it was earlier, since you have only changed the structure of these 3 functions